benchmark/citation/train_eval.py

```diff
@@ -96,10 +97,18 @@ def run_train(dataset, model, runs, epochs, lr, weight_decay, early_stopping,
         with torch_profile():
             train(model, optimizer, data)
 
+    if use_compile:
```

Comment:
Thanks for the update. I am still not sure why we added it this way. I would expect that the `compile` flag converts the model to a compiled model at the start of this `train` function, rather than making a separate training run. I think I would prefer this, but if you have strong opjections we can also merge it in the current state.

Reply:
I understand your point. I was influenced by `benchmark.py` to compare eager mode and compiled mode in the same run.

I will remove this logic among `kernel`, `citation` and `points`.